NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -1486,11 +1489,12 @@ void HTauTauNtuplizer::beginJob(){
     myTree->Branch("PUNumInteractions",&_PUNumInteractions,"PUNumInteractions/I");  
     myTree->Branch("daughters_genindex",&_daughters_genindex);
     myTree->Branch("MC_weight",&_MC_weight,"MC_weight/F");
-    myTree->Branch("MC_weight_scale_muF0p5",&_MC_weight_scale_muF0p5,"MC_weight_scale_muF0p5/F");
-    myTree->Branch("MC_weight_scale_muF2",&_MC_weight_scale_muF2,"MC_weight_scale_muF2/F");
-    myTree->Branch("MC_weight_scale_muR0p5",&_MC_weight_scale_muR0p5,"MC_weight_scale_muR0p5/F");
-    myTree->Branch("MC_weight_scale_muR2",&_MC_weight_scale_muR2,"MC_weight_scale_muR2/F");
-    myTree->Branch("MC_weight_PSWeight0",&_MC_weight_PSWeight0,"MC_weight_PSWeight0/F");
+
+	myTree->Branch("MC_QCDscale", &_MC_QCDscale, "MC_QCDscale/F");
+	myTree->Branch("MC_pdf", &_MC_pdf, "MC_pdf/F");
+	myTree->Branch("MC_astrong", &_MC_astrong, "MC_astrong/F");
+
+	myTree->Branch("MC_weight_PSWeight0",&_MC_weight_PSWeight0,"MC_weight_PSWeight0/F");
```

Comment:
Question originating from my ignorance: will we need these PSWeights with the new implementation of the uncertainty scheme?

Reply:
Not AFAIK, but the communication via CMS Talk is so far extremely hard to follow, so I cannot guarantee.

Reply:
Ok, thanks for checking! And how did we end up with 100 new branches? What do they contain?

Reply:
So we need to add 50 systematic uncertainties ti the analysis... ?

Reply:
We only have to add 3 systematics, but they require all the 108 branches above. Details were clarified in private.